### PR TITLE
UI-91 Add nested anchor tag styles to the Typography component

### PR DESCRIPTION
**Pull request by Sonicrida**
Closes UI-91

### CODE REVIEW

File: src/atoms/Typography/Typography.tsx [hunk 1]
```diff
@@ -5,6 +5,15 @@ export const Typography = styled.p`
   color: ${props => props.theme.text};
   font-family: Lato, sans-serif;
   line-height: 1.5;
+
+  a {
+    color: ${props => props.theme.link};
```
Comment (nickmccurdy) [inline src/atoms/Typography/Typography.tsx]: I think we should use the "links in line" colors here and make it bold, thoughts? We could potentially have a different component for standalone links, but then we'd need a dedicated link component again. https://app.zeplin.io/project/5b0334f5e91e8c481645ad56/screen/5bb23e07eff8754297471dff

File: src/atoms/Typography/Typography.test.tsx [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+  rerender(
+    <Typography>
+      <a href="https://example.com/">Link Text</a>
+    </Typography>,
+  );
+  expect(getByText('Link Text')).toHaveTextContent('Link Text');
+  const link = getByText('Link Text');
+  expect(link).toHaveTextContent('Link Text');
+  expect(link).toHaveAttribute('href', 'https://example.com/');
+});
```
[the remark this answers is not in the document]
Comment (Sonicrida) [reply src/atoms/Typography/Typography.test.tsx]: That makes sense! I'll remove the tests.

File: package.json [hunk 3]
```diff
@@ -155,7 +155,7 @@
       "declaration-no-important": true,
       "font-family-name-quotes": "always-where-recommended",
       "font-weight-notation": "named-where-possible",
-      "max-nesting-depth": 1,
+      "max-nesting-depth": 2,
```
[the remark this answers is not in the document]
Comment (Sonicrida) [reply package.json]: Gotcha. I'll change it back and use the temporary disable suggested below.

File: src/atoms/Typography/Typography.tsx [hunk 4]
```diff
@@ -5,6 +5,15 @@ export const Typography = styled.p`
...
+    text-decoration: none;
+
```
[the remark this answers is not in the document]
Comment (Sonicrida) [reply src/atoms/Typography/Typography.tsx]: I'll push up these changes

File: src/atoms/Typography/Typography.tsx [hunk 5]
```diff
@@ -4,7 +4,18 @@ import styled from '../../styled-components';
 export const Typography = styled.p`
   color: ${props => props.theme.text};
   font-family: Lato, sans-serif;
+  font-weight: bold;
```
Comment (nickmccurdy) [inline src/atoms/Typography/Typography.tsx]: I think this should be inside `a {}`, it's making all Typography children bold.

Comment (Sonicrida) [reply src/atoms/Typography/Typography.tsx]: Good call!

**Pull request merged**